src/Domain.LinnApps/Requisitions/RequisitionManager.cs

```diff
@@ -684,6 +684,11 @@ public async Task<RequisitionHeader> Validate(
                     throw new CreateRequisitionException($"PO {document1Number} does not exist!");
                 }
                 
+                if (po.DocumentType != "PO")
```

Comment:
i think this might break GISTPO for COs and ROs

Reply:
Ok I can see there RO and CO GIST POs so remove that clause